Index: src/main/resources/com/example/joncdstore/Login.fxml
===================================================================
diff --git a/src/main/resources/com/example/joncdstore/Login.fxml b/src/main/resources/com/example/joncdstore/Login.fxml
deleted file mode 100644
--- a/src/main/resources/com/example/joncdstore/Login.fxml	(revision 6d6be9e1581a1a0ddaa000bb2ae757ec299242b9)
+++ /dev/null	(revision 6d6be9e1581a1a0ddaa000bb2ae757ec299242b9)
@@ -1,47 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.text.Font?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="652.0" prefWidth="879.0" styleClass="main" stylesheets="@../../../FXstyle.css" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-   <children>
-      <GridPane alignment="CENTER" layoutX="237.0" layoutY="33.0" prefHeight="509.0" prefWidth="428.0">
-         <columnConstraints>
-            <ColumnConstraints hgrow="ALWAYS" maxWidth="400.0" minWidth="10.0" prefWidth="171.0" />
-            <ColumnConstraints hgrow="ALWAYS" maxWidth="400.0" minWidth="10.0" prefWidth="257.0" />
-         </columnConstraints>
-         <rowConstraints>
-            <RowConstraints maxHeight="439.0" minHeight="10.0" prefHeight="413.0" vgrow="SOMETIMES" />
-            <RowConstraints maxHeight="439.0" minHeight="0.0" prefHeight="55.0" vgrow="SOMETIMES" />
-            <RowConstraints maxHeight="439.0" minHeight="0.0" prefHeight="41.0" vgrow="SOMETIMES" />
-         </rowConstraints>
-         <children>
-            <Text id="rainbow-text" fx:id="Title" strokeType="OUTSIDE" strokeWidth="0.0" style="-fx-font-family: &quot;Brush Script MT&quot;; -fx-font-size: 70;" text="C D W O R L D" textAlignment="CENTER" wrappingWidth="456.359375">
-               <font>
-                  <Font size="48.0" />
-               </font>
-            </Text>
-            <Label fx:id="UsernameLabel" alignment="TOP_RIGHT" prefHeight="30.0" prefWidth="168.0" style="-fx-font-size: 20; -fx-font-weight: Bold;" text="Username" textFill="#222121" GridPane.rowIndex="1">
-               <font>
-                  <Font name="System Bold" size="14.0" />
-               </font></Label>
-            <Label fx:id="PasswordLabel" alignment="CENTER_RIGHT" prefHeight="30.0" prefWidth="167.0" style="-fx-font-size: 20; -fx-font-weight: bold;" text="Password" textAlignment="CENTER" textFill="#1e1e1e" GridPane.rowIndex="2" />
-            <TextField fx:id="PasswordField" maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="25.0" prefWidth="184.0" promptText="Enter password" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-            <TextField fx:id="usernameField" maxHeight="25.0" maxWidth="183.0" minHeight="20.0" minWidth="20.0" prefHeight="25.0" prefWidth="183.0" promptText="Enter username" GridPane.columnIndex="1" GridPane.rowIndex="1">
-               <GridPane.margin>
-                  <Insets />
-               </GridPane.margin>
-            </TextField>
-         </children>
-      </GridPane>
-      <Button layoutX="412.0" layoutY="551.0" mnemonicParsing="false" prefHeight="25.0" prefWidth="96.0" styleClass="login-but" text="Login" />
-   </children>
-</AnchorPane>
Index: target/classes/com/example/joncdstore/Login.fxml
===================================================================
diff --git a/target/classes/com/example/joncdstore/Login.fxml b/target/classes/com/example/joncdstore/Login.fxml
deleted file mode 100644
--- a/target/classes/com/example/joncdstore/Login.fxml	(revision 6d6be9e1581a1a0ddaa000bb2ae757ec299242b9)
+++ /dev/null	(revision 6d6be9e1581a1a0ddaa000bb2ae757ec299242b9)
@@ -1,47 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.text.Font?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="652.0" prefWidth="879.0" styleClass="main" stylesheets="@../../../FXstyle.css" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-   <children>
-      <GridPane alignment="CENTER" layoutX="237.0" layoutY="33.0" prefHeight="509.0" prefWidth="428.0">
-         <columnConstraints>
-            <ColumnConstraints hgrow="ALWAYS" maxWidth="400.0" minWidth="10.0" prefWidth="171.0" />
-            <ColumnConstraints hgrow="ALWAYS" maxWidth="400.0" minWidth="10.0" prefWidth="257.0" />
-         </columnConstraints>
-         <rowConstraints>
-            <RowConstraints maxHeight="439.0" minHeight="10.0" prefHeight="413.0" vgrow="SOMETIMES" />
-            <RowConstraints maxHeight="439.0" minHeight="0.0" prefHeight="55.0" vgrow="SOMETIMES" />
-            <RowConstraints maxHeight="439.0" minHeight="0.0" prefHeight="41.0" vgrow="SOMETIMES" />
-         </rowConstraints>
-         <children>
-            <Text id="rainbow-text" fx:id="Title" strokeType="OUTSIDE" strokeWidth="0.0" style="-fx-font-family: &quot;Brush Script MT&quot;; -fx-font-size: 70;" text="C D W O R L D" textAlignment="CENTER" wrappingWidth="456.359375">
-               <font>
-                  <Font size="48.0" />
-               </font>
-            </Text>
-            <Label fx:id="UsernameLabel" alignment="TOP_RIGHT" prefHeight="30.0" prefWidth="168.0" style="-fx-font-size: 20; -fx-font-weight: Bold;" text="Username" textFill="#222121" GridPane.rowIndex="1">
-               <font>
-                  <Font name="System Bold" size="14.0" />
-               </font></Label>
-            <Label fx:id="PasswordLabel" alignment="CENTER_RIGHT" prefHeight="30.0" prefWidth="167.0" style="-fx-font-size: 20; -fx-font-weight: bold;" text="Password" textAlignment="CENTER" textFill="#1e1e1e" GridPane.rowIndex="2" />
-            <TextField fx:id="PasswordField" maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="25.0" prefWidth="184.0" promptText="Enter password" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-            <TextField fx:id="usernameField" maxHeight="25.0" maxWidth="183.0" minHeight="20.0" minWidth="20.0" prefHeight="25.0" prefWidth="183.0" promptText="Enter username" GridPane.columnIndex="1" GridPane.rowIndex="1">
-               <GridPane.margin>
-                  <Insets />
-               </GridPane.margin>
-            </TextField>
-         </children>
-      </GridPane>
-      <Button layoutX="412.0" layoutY="551.0" mnemonicParsing="false" prefHeight="25.0" prefWidth="96.0" styleClass="login-but" text="Login" />
-   </children>
-</AnchorPane>
